utils/spdx/src/test/kotlin/SpdxSimpleLicenseMappingTest.kt

```diff
@@ -57,6 +57,23 @@ class SpdxSimpleLicenseMappingTest : WordSpec({
                 if (license.id.endsWith("-only")) key should containADigit()
             }
         }
+
+        "contain equal digits omitting trailing zeros" {
+            val exceptions = mapOf(
```

Comment:
There are license IDs which have dates in their identifier, e.g. some `LicenseRef-scancode-*` licenses. 
There may be further cases, where this heuristic fails.

Reply:
I'd be fine with adding anything that start with `LicenseRef-` to the list of exceptions. But we don't have such mappings currently anyway.